content/ru/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -0,0 +1,180 @@
+
+Google открыл доступ к исходному коду Kubernetes в 2014 году. Kubernetes создан после [полутора десятилетий опыта, который Google обрёл при работе с масштабированием рабочей нагрузки](https://research.google.com/pubs/pub43438.html), объединённым с лучшими идеями и практиками сообщества.
+
+## Зачем мне Kubernetes и что можно сделать с его помощью
+
+В Kubernetes реализовано большое количество возможностей. Можно обобщить их следующим образом:
+
+- платформа для контейнеров
+- платформа для микросервисов
+- портативная облачная платформа
+а также многое другое.
+
+Kubernetes предоставляет **контейнеро-ориентированное** окружение для управления. Оно оркестрирует вычислительную и сетевую инфраструктуру, а также инфраструктуру хранения от имени рабочей нагрузки от пользователей. Это позволяет добиться большей простоты платформы как сервиса (PaaS) не теряя гибкости инфраструктуры как сервиса (IaaS), а также даёт возможность портативности среди поставщиков инфраструктуры. 
+
+## Почему Kubernetes это платформа
+
+Несмотря на то, что у Kubernetes большой спектр возможностей, всегда есть сценарии, требующие новых функций. Последовательности действий, специфичные для конкретного приложения, могут быть оптимизированы, чтобы ускорить скорость разработки. Специализированная оркестрация, которая вначале подходила, при масштабировании может требовать надёжной автоматизации. По этим причинам Kubernetes создавался как платформа для создания экосистемы из компонентов и инструментов для облегчения развёртывания, масштабирования и управления приложениями.
+
+[Лейблы](/docs/concepts/overview/working-with-objects/labels/) позволяют пользователям организовывать ресурсы по собственному желанию. [Аннотации](/docs/concepts/overview/working-with-objects/annotations/)
+дают возможность пользователям сопровождать ресурсы дополнительной информацией, чтобы упростить их рабочие процессы и предоставить простой способ управления инструментами для проверки состояния.
+
+Также [панель управления Kubernetes](/docs/concepts/overview/components/) построена на том же [API](/docs/reference/using-api/api-overview/), которое доступно разработчикам и пользователям. Пользователи могут сами разрабатывать контролдеры, например [планировщики](https://github.com/kubernetes/community/blob/{{< param "githubbranch" >}}/contributors/devel/scheduler.md), [с собственным 
+API](/docs/concepts/api-extension/custom-resources/), которые могут быть предназначены для [инструментов командной строки](/docs/user-guide/kubectl-overview/) общего назначения.
+
+Такая 
+[конструкция](https://git.k8s.io/community/contributors/design-proposals/architecture/architecture.md)
+позволила создать несколько других систем на базе Kubernetes.
+
+## What Kubernetes is not
+
+Kubernetes is not a traditional, all-inclusive PaaS (Platform as a
+Service) system. Since Kubernetes operates at the container level
```

Comment:
Could you translate this chunk of text?

Reply:
@remyleone do you mean that we need to translate all the text remaining in this file? I thought we can do this after initializing the translation, because this PR is already large and complicated for review.

Reply:
That's fine to translate this content later. Could you take the content out of this PR and start another PR with it? I think it makes sense to have PR to merge only Russian content but not a copy of English. It is better to stay consistent.
@zacharysarah Could you confirm?

Reply:
Fair enough, I'll do it when it's confirmed.

Reply:
> I think it makes sense to have PR to merge only Russian content but not a copy of English.

That also makes sense to me.

Reply:
It's done, I’ll create another PR after the translation is initialized, then there will be a folder `content/ru`.

Reply:
Perfect :)

Reply:
So we ready for merge? :)